modules/kmm-replacing-in-tree-modules-with-out-of-tree-modules.adoc

```diff
@@ -10,7 +10,7 @@ You can use Kernel Module Management (KMM) to build kernel modules that can be l
 
 Dynamically loaded modules include in-tree modules and out-of-tree (OOT) modules. In-tree modules are internal to the Linux kernel tree, that is, they are already part of the kernel. Out-of-tree modules are external to the Linux kernel tree. They are generally written for development and testing purposes, such as testing the new version of a kernel module that is shipped in-tree, or to deal with incompatibilities.
 
-Some modules loaded by KMM could replace in-tree modules already loaded on the node. To unload an in-tree module before loading your module, set the `.spec.moduleLoader.container.inTreeModuleToRemove` field. The following is an example for module replacement for all kernel mappings:
+Some modules that are loaded by KMM could replace in-tree modules that are already loaded on the node. To unload in-tree modules before loading your module, set the value of `.spec.moduleLoader.container.inTreeModulesToRemove` accordingly. The following example demonstrates module replacement for all kernel mappings:
```

Comment:
Like max mentioned here [1], ISG wants you to qualify fields, etc. Since this used to say "the `.spec.moduleLoader.container.inTreeModuleToRemove` field", I would keep that in your new version.

[1] https://github.com/openshift/openshift-docs/pull/77130#discussion_r1633495860

```suggestion
Some modules that are loaded by KMM could replace in-tree modules that are already loaded on the node. To unload in-tree modules before loading your module, set the value of the `.spec.moduleLoader.container.inTreeModulesToRemove` field accordingly. The following example demonstrates module replacement for all kernel mappings:
```

If you care to, you could also fix it in line 27 where `inTreeModulesToRemove` is also mentioned. But that one's not as critical since it was already like that.

Reply:
Added "field" back.

Reply:
@StephenJamesSmith Don't forget the  "the" in front as well

Reply:
whoops. added.